docs/wiki/ParameterFileTokens.md

```diff
@@ -0,0 +1,172 @@
+
+![paramFileTokenSetKeyVault](../media/paramFileTokenSetKeyVault.jpg)
+
+1- The user can create the Secret called `PLATFORM_KEYVAULT`. This must be a unique Key Vault name across Azure.
+
+  > The parameter files can now be tokenized as per required value. Then token must be in the format of `<<tokenName>>`. Either locally or remotely on the Key Vault.
+
+2- The user then triggers the [Dependency Workflow](../../.github/workflows/platform.dependencies.yml) to instantiate the Platform Token Key Vault.
+
+  > To customize the Key Vault Configuration, you can modify the [Platform Key Vault Parameter File](../../utilities/dependencies/Microsoft.KeyVault/vaults/parameters/platform.parameters.json).
+
+3- The Platform Key Vault is then deployed using the Key Vault Module and the dependency [parameter file]((../../utilities/dependencies/Microsoft.KeyVault/vaults/parameters/platform.parameters.json).).This Key Vault uses [Azure AD RBAC](https://docs.microsoft.com/en-us/azure/key-vault/general/rbac-guide?tabs=azure-cli) for the permissions instead of [access policies](https://docs.microsoft.com/en-us/azure/key-vault/general/assign-access-policy?tabs=azure-portal). It will automatically provide the Deployment Service Principal with the [Key Vault Administrator](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#key-vault-administrator) Role so that it can read and set secrets.
+
+  > You need to have permissions to read and create secrets in the provisioned Key Vault so that you can upload custom tokens. The [Key Vault Secrets Officer](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#key-vault-secrets-officer) role is an example that allows you to perform data plane operations on the Key Vault.
+
+4- Once the Key Vault is deployed, the 'Set Local Tokens in Key Vault' task optionally runs to sync Local Custom Parameter File Tokens stored in the [Settings.json](../../settings.json)
+
+  > You can disable the sync process from Local Tokens (in Source Control) to Remote Tokens (in Key Vault) by modifying the `keyVaultSecretNamePrefix` property in the [Settings.json](../../settings.json) file and setting it to `false`.</br>
+  > This step/task is optional and can be used to share tokens across different repositories.
+
+### Get Parameter File Tokens
+
+ ---
+**Note**: This Step is always enabled even if you do not use Local/Remote Custom Parameter File Tokens. Default Tokens are always required and are Pester Tested if they do not exist in a Parameter file where they should.
+
+---
+
+The below diagram illustrates the Token Replacement Functionality via the [Validate](../../.github/actions/templates/validateModuleDeploy/action.yml) and [Deploy](../../actions/../.github/actions/templates/deployModule/action.yml) Actions/Templates.
```

Comment:
link issue... also, what are you trying to do here with the link? up one level, up one level, into actions, up one level, then .github? I might be missing the point here :D

Reply:
oh that's probably a find/replace issue.. good pickup !

Reply:
Fixed